src/Appwrite/Utopia/Response/Model/UsageFunction.php

```diff
@@ -16,58 +16,52 @@ public function __construct()
                 'default' => '',
                 'example' => '30d',
             ])
-            ->addRule('executionsTotal', [
+            ->addRule('deployments', [
```

Comment:
We use `documentsCount` and `collectionsCount` in the database usage and only `deployments`, `builds` etc in the functions usage. We need to maintain consistency